OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -151,9 +192,10 @@ open class OptimoveConfigBuilder: NSObject {
                 inAppConsentStrategy: _inAppConsentStrategy,
                 inAppDeepLinkHandlerBlock: _inAppDeepLinkHandlerBlock,
                 pushOpenedHandlerBlock: _pushOpenedHandlerBlock,
-                _pushReceivedInForegroundHandlerBlock: _pushReceivedInForegroundHandlerBlock,
+                pushReceivedInForegroundHandlerBlock: _pushReceivedInForegroundHandlerBlock,
                 deepLinkCname: _deepLinkCname,
                 deepLinkHandler: _deepLinkHandler,
+                objCdeepLinkHandler: _objCdeepLinkHandler,
```

Comment:
I'm not sure we need to pass this down into OptimobileConfig, it looks like it just needs held at the OptimoveConfigBuilder level? (the closure is passed from _deepLinkHandler)

Reply:
this code seems to not exist any more